### ISSUE TITLE
Errors when running on Windows

### ISSUE THREAD

**Issue by nhrones** [webview/webview_deno#179]
I get errors when running on Windows.
Seems the problem is incorrect code on @jsr.
The ffi code on jsr tests for 'window' rather than 'globalThis'.

The GitHub version of /src/ffi.ts uses globalThis and works correctly.
```ts
if (Deno.build.os === "windows") {
  if ((self as never)["globalThis"]) {
```

The version of /src/ffi.ts on @jsr is still testing for 'window'
```ts
if (Deno.build.os === "windows") {
  if ((self as never)["window"]) {
```

The 'globalThis' version runs without errors.

Does the @jsr need to be updated?

Should I use a GitHub or Deno/x link for 0.8.0 ?

**Comment by eliassjogreen** [comment]
Hey! Just released a new x version 0.8.1 which should fix it. The JSR version does indeed need to be updated, doing that now.

**Comment by nhrones** [comment]
Thanks!  Very fast response!
